docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-geopartition.md

```diff
@@ -91,14 +92,15 @@ Fault tolerance is applied to all regions in the cluster, including those added
 
 **Regions** - For each region, choose the following:
 
-- the [region](../../create-clusters-overview/#cloud-provider-regions) where the nodes will be located.
-- the VPC in which to deploy the nodes. Only VPCs using the selected cloud provider and available in the selected region are listed. For multi-region GCP clusters, the same VPC is used for all regions. VPCs must be created before deploying the cluster. Refer to [VPC networking](../../cloud-vpcs/).
-- the number of nodes to deploy in the regions; each region has the same number of nodes.
+- [region](../../create-clusters-overview/#cloud-provider-regions) where the nodes will be located.
+- VPC in which to deploy the nodes. Only VPCs using the selected cloud provider and available in the selected region are listed. For multi-region GCP clusters, the same VPC is used for all regions. VPCs must be created before deploying the cluster. Refer to [VPC networking](../../cloud-vpcs/).
+- number of nodes to deploy in the regions; each region has the same number of nodes.
```

Comment:
>each region has the same number of nodes.
This is not true for asymmetric geo clusters. You can specify different # of nodes per region there.